collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java

```diff
@@ -379,6 +385,16 @@ else if (mediaLoadData.dataType == DATA_TYPE_MEDIA &&
             this.drmType = drmType;
         }
 
+        if(mediaLoadData.trackFormat != null
```

Comment:
Small nitpick but i feel like the onLoadCompleted callback is already doing too much here.
Might make sense to refactor it a bit and break it into different methods that are named after the different things that are happening (manifest, drm, videosegment)